src/components/shared/Pagination/Pagination.test.js

```diff
@@ -0,0 +1,142 @@
+  });
+
+  it('should not show next page element when current page is the last one', () => {
+    //Act
+    const { container } = render(
+      <BrowserRouter>
+        <Pagination currentPage={100} pagesCount={100} urlToGo={'https://domain.com/section?'} />
+      </BrowserRouter>,
+    );
+
+    // Asserts
+    expect(container.querySelector('.icon-arrow-next')).not.toBeInTheDocument();
+  });
+
+  it('should show ... to slide to left when selected page is not the first one', () => {
+    //Act
+    const { container } = render(
+      <BrowserRouter>
+        <Pagination currentPage={3} pagesCount={100} urlToGo={'https://domain.com/section?'} />
+      </BrowserRouter>,
+    );
+
+    // Asserts
+    expect(container.querySelector('#pag-point-left')).toBeInTheDocument();
+  });
+
+  it('should show ... to slide to right when selected page is not one of the last 5 pages', () => {
+    //Act
+    const { container } = render(
+      <BrowserRouter>
+        <Pagination currentPage={90} pagesCount={100} urlToGo={'https://domain.com/section?'} />
+      </BrowserRouter>,
+    );
+
+    // Asserts
+    expect(container.querySelector('#pag-point-right')).toBeInTheDocument();
+  });
+
+  it('should not show ... to slide to left when selected page is the first one', () => {
+    //Act
+    const { container } = render(
+      <BrowserRouter>
+        <Pagination currentPage={1} pagesCount={100} urlToGo={'https://domain.com/section?'} />
+      </BrowserRouter>,
+    );
+
+    // Asserts
+    expect(container.querySelector('#pag-point-left')).not.toBeInTheDocument();
+  });
+
+  it('should not show ... to slide to right when selected page is one of the last 5 pages', () => {
+    //Act
+    const { container } = render(
+      <BrowserRouter>
+        <Pagination currentPage={96} pagesCount={100} urlToGo={'https://domain.com/section?'} />
+      </BrowserRouter>,
+    );
+
+    // Asserts
+    expect(container.querySelector('#pag-point-right')).not.toBeInTheDocument();
+  });
+
+  it('should not show any ... neither to slide to right neither to slide to left if page count is 5', () => {
+    //Act
+    const { container } = render(
+      <BrowserRouter>
+        <Pagination currentPage={1} pagesCount={5} urlToGo={'https://domain.com/section?'} />
+      </BrowserRouter>,
+    );
+
+    // Asserts
+    expect(container.querySelector('#pag-point-right')).not.toBeInTheDocument();
+    expect(container.querySelector('#pag-point-left')).not.toBeInTheDocument();
+  });
+
+  it('should not show pagination if there is no pages', () => {
+    //Act
+    const { container } = render(
+      <BrowserRouter>
+        <Pagination currentPage={1} pagesCount={0} urlToGo={'https://domain.com/section?'} />
+      </BrowserRouter>,
+    );
+
+    // Asserts
+    expect(container.querySelector('.dp-pagination')).not.toBeInTheDocument();
+  });
+
+  it('should show pagination if there is at least one page', () => {
```

Comment:
I'm not so sure about this, why would we want a pagination if you can't paginate over another page? 🤔

Reply:
< 1 > 🤔

Reply:
Yes, is a little weird but another test exists, for invoices list page, that specify that the pagination element should be shown when there are information to show. I didn't want to have problem with that specification.

Reply:
Perhaps the implementation is not so good. This is the great thing about tests, they let us think more about possible cases...

Reply:
Yes, I am agree with you. I can change the rules for this other test.

Reply:
Perhaps we should ask for ux opinion @santiagopaolucci what do you think?

Reply:
As we talk in slack with @santiagopaolucci and @mdirago we will show pagination element even if there is only one page.